src/previewSupport/handleConsoleLogs.js

```diff
@@ -0,0 +1,32 @@
+import channel from './channel';
+
+function notifyChannel(args) {
+  if (args.length > 0) {
+    const output = args.map(arg => JSON.stringify(arg)).join(' ');
+    channel.notify({
+      method: 'log',
+      params: {output},
+    });
+  }
+}
+
+export default function handleConsoleLogs() {
+  const consoleFunctions = ['log', 'info', 'warn', 'error'];
+
+  // eslint-disable-next-line no-console
+  if (console.debug) {
+    consoleFunctions.push('debug');
+  }
+
+  consoleFunctions.forEach((functionName) => {
+    // eslint-disable-next-line no-console
+    const browserFunction = console[functionName].bind(console);
+    // eslint-disable-next-line prefer-reflect
+    Object.defineProperty(console, functionName, {
```

Comment:
Did `Reflect` cause a problem here? We might need to pull the Babel polyfill into the preview support script…

Reply:
I saw Reflect wasn't supported back to Chrome 22 and the other preview frame code also disabled prefer-reflect - do you want me to make this Babel polyfill change in this PR?